packages/menu/src/Menu.spec.tsx

```diff
@@ -125,7 +135,8 @@ describe('packages/menu', () => {
         const button = getByRole('button');
         userEvent.click(button);
         const menu = getByTestId(menuTestId);
-        userEvent.type(menu, '{esc}');
+
```

Comment:
can we add the same test for when usePortal is false?

Reply:
there is a false test, its just not showing up in this PR since i didn't touch that line

Reply:
`test('Returns focus to trigger {usePortal: false}', async () => {`

Reply:
ah ok